examples/multi-container/ruby/README.md

```diff
@@ -98,6 +98,8 @@ spec:
 
              # Ensure the port number on the --port argument matches the value of the DB_PORT env var on the my-app container.
              - "--port=5432"
+            # Instance URIs follow the format 
+            # projects/PROJECT_ID/locations/REGION_ID/clusters/CLUSTER_ID/instances/INSTANCE_ID
```

Comment:
would `REGION_ID`, `CLUSTER_ID`, and `INSTANCE_ID` be better off as `REGION_NAME`, `CLUSTER_NAME`, and `INSTANCE_NAME`. If these are referred to as IDs in the docs then we can leave as is.

Reply:
I copied this straight from the docs!